MHRA/products
# Update search indexes & move to MHRA Azure.

### Name of the story

N/A

![](https://media0.giphy.com/media/l0HlGTJmgaz2nVdHW/giphy.gif?cid=790b7611d50b6ab58aacf54018fd69ad28b14279008932ee&rid=giphy.gif)

### Acceptance Criteria

- [x] Index definition now contains 'facets' field;
- [x] Indexer definition creates facets from JSON array;
- [x] Review fields and make them facetable / filterable where sensible;
- [x] Development site now points to MHRA Azure instead of RB Azure.

### Testing information

1. Navigate to the site and open dev tools (F12).
2. Go to the network tab.
3. Make a search.
4. Look in the results for 'facets', they should be an array of strings.

### Pre-merge Checklist
